docs/wooooo.hbs

```diff
@@ -1,5 +1,6 @@
 ---
 title: wooooo
+draft: true
 ---
 {{#> page}}
   {{#*inline "title"}}{{title}}{{/inline}}
```

Comment:
That's currently the only page showing how to include a snippet and a demo in a handlebars page